<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>351</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Построение картографической сетки</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Границы рамки</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Север:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_bnd_north">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Юг:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_bnd_south">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_bnd_east">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Восток:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Запад:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_bnd_west">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>136</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_unit_north">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_unit_south">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_unit_east">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_unit_west">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_using_obj">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Использовать Bound выделенного объекта</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Прекция</string>
   </property>
   <widget class="QLabel" name="lb_curent_prj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_change_coodsys">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>15</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Выбрать проекцию">Изменить проекцию</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>321</width>
     <height>216</height>
    </rect>
   </property>
   <property name="title">
    <string>Оформление</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>306</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Шаг сетки</string>
    </property>
    <widget class="QLineEdit" name="ln_step_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_unti_step_line">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>20</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_add_points">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>70</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сглаживание</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>дополнительные узлы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_update_step">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>15</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Пересчитать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>45</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Число линий</string>
     </property>
    </widget>
    <widget class="QLabel" name="count_lines">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>45</y>
       <width>66</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labestep_grad">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_grag">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>150</y>
      <width>166</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Формат градусных меток</string>
    </property>
    <widget class="QComboBox" name="cm_format_label_grd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>135</y>
      <width>136</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Стиль</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>15</y>
       <width>51</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="wg_style_line">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>331</width>
     <height>46</height>
    </rect>
   </property>
   <property name="title">
    <string>Результирующая таблица</string>
   </property>
   <widget class="QLineEdit" name="ln_edit_out_tab">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>286</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_saveAs">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>15</y>
      <width>32</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Изменить файл</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>510</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
